core/backend/Data/LegacyHandler/PresetDataHandlers/HistoryTimelineDataHandler.php

```diff
@@ -277,7 +277,7 @@ public function fetch(
         $auditUnionQuery = $this->queryAuditInfo($parentBean);
         $combinedQuery = $historyUnionQuery .
             ' UNION ALL ' . $activitiesUnionQuery .
-            ' UNION ALL ' . $auditUnionQuery .
+            ' UNION ALL (' . $auditUnionQuery . ')'
```

Comment:
Hi @eggsurplus,

You appear to be missing a `.` after you have added in the quote.